### ISSUE TITLE
Deberian eliminarse las referencias internas del grupo de discord?

### ISSUE THREAD

**Issue by leopolanco** [JGPenaB/Pasante-bot#24]
Hablo de los comandos como test, bayke, etc. 
Preparandonos para una internacionalizacion y lanzamiento al mercado piensan que se es necesario eliminar estas referencias? 
Para hacer de pasante un poco mas inclusivo y profesional

**Comment by leopolanco** [comment]
@cedaesca @darioxlz @JGPenaB

**Comment by cedaesca** [comment]
Sí, ya esto se había discutido y es como se hará en la nueva versión de pasante en Typescript

**Comment by darioxlz** [comment]
Con un panel administrativo podemos administrar que comandos estan disponibles en cada servidor. Con respecto al a i18n se pueden crear diccionarios en un JSON tipo es.json o en.json, y una funcion global que traduzca la key que le pases. Deberia llevarse una tabla con la metadata del servidor en donde esta pasante, como por ejemplo, el idioma que maneja en ese server en particular.

**Comment by cedaesca** [comment]
Manejar estas sugerencias en https://github.com/pasante-bot

**Comment by leopolanco** [comment]
Estoy en desacuerdo con cedaesca, ese repositorio esta abandonado y su ultima actualizacion es mas antigua que la de este repositorio. 

Reabro, este es el lugar correcto para sugerencias.
